src/server/pachyderm_test.go

```diff
@@ -10534,3 +10536,97 @@ func TestMissingSecretFailure(t *testing.T) {
 		return nil
 	})
 }
+
+type rowCountResults struct {
+	IdxScan    int    `json:"idx_scan"`
+	NLiveTup   int    `json:"n_live_tup"`
+	RelName    string `json:"relname"`
+	SchemaName string `json:"schemaname"`
+	SeqScan    int    `json:"seq_scan"`
+}
+
+func TestPostgresStats(t *testing.T) {
+	if testing.Short() {
+		t.Skip("Skipping integration tests in short mode")
+	}
+
+	repoName := tu.UniqueString("TestPostgressStats-repo")
+	branchName := "master"
+	numCommits := 5
+	c, _ := minikubetestenv.AcquireCluster(t)
+
+	createTestCommits(t, repoName, branchName, numCommits, c)
+
+	buff := &bytes.Buffer{}
+	filter := &debug.Filter{
+		Filter: &debug.Filter_Database{Database: true},
+	}
+
+	require.NoError(t, c.Dump(filter, 100, buff))
+
+	files := untar(t, buff)
+
+	for fileName := range files {
+		if strings.Contains(fileName, ".error") {
+			t.Fatalf("error returned from %s", fileName)
```

Comment:
I suppose for consistency this should try to use `require`, but that only seems to have Matches, NoneEquals, and OneOfMatches... not NoneMatches.

General thoughts on assertion libraries though: https://github.com/golang/go/wiki/TestComments#assert-libraries

This is basically why they're not recommended... here you are in a test file that only uses assert libraries, but now you have to go off and write a general assertion function instead of a simple loop and == statement.  For that reason, I don't think you should go write that, but be aware of `require` when working in this codebase.